join.md

```diff
@@ -6,42 +6,39 @@ permalink: /subscribe/
 
 # Join us
 
-Our Slack is governed by the principles and rules in our [Community Guide](/community-guide). By joining, you agree to follow them.
+1. Fill out this form to subscribe to our newsletter.
+2. If eligible, you will be invited to our Slack.
+3. You agree to the [Community Guide](/community-guide) by filling the form.
 
-<h3 class="marg-b-3">Please provide the following:</h3>
+<h3>Please provide the following:</h3>
 <form class="join-form" method="POST" target="_blank" class="marg-b-4" data-netlify="true" action="/welcome" netlify-honeypot="bot-field">
   <label style="display:none">
     Don’t fill this out if you’re human: <input name="bot-field" />
   </label>
-  <label class="marg-b-3" for="email">
-    <div><b>Email:</b></div>
+  <label for="name">
+    <div><b>Your full name</b></div>
+    <input id="name" type="text" required name="name">
+  </label>
+  <label for="email">
+    <div><b>Email</b></div>
     <input id="email" type="email" required name="email">
   </label>
-  <label class="marg-b-3" for="name">
-    <div><b>Name:</b></div>
-    <input id="name" type="text" required name="name">
+  <label for="social">
+    <div><b>Social media link #1</b></div>
+    <input placeholder="LinkedIn, Bluesky, Twitter, etc.." id="social" required type="text" name="social_media_1" title="enter a valid url">
+    <div><b>Social media link #2</b></div>  
+    <input placeholder="GitHub, Instagram, Blog etc.." type="text" required name="social_media_2" title="enter a valid url">
   </label>
-  <label class="marg-b-3" for="social">
-    <div class="marg-b-2">
-      <b>Please provide two links to your social media.</b>
```

Comment:
I would add back some of the explanation for why we want social media links, it was helpful when I filled the form out myself